### PR TITLE
Create contact form section validate inputs formspree service & footer section *corrections applied

### PR DESCRIPTION
**Pull request by evillaz**
This branch's purpose was to:
- Use flexbox for the structure design
- Implement button behavior(hover, active)
- Create a contact form section
   -  Added name, email, text and submit button elements.
   - validate the length of the fields
   - Submit data using formspree service
- Create a footer section
   - Added social media icons and background shapes.
  *****************
 *Modified "section" as a flex container for the "p" and "form" elements
 *Corrected Textarea sizing on CSS
 *Corrected coloring in social media icons
 *Corrected ul syntax for semantically correct structure

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -351,6 +351,61 @@ <h4 class="featureText">Terminal</h4>
...
+                <li class="formElement">
+                  <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+                </li>
+                <li id="submitForm">
+                  <button class="button" id="subBut" type="submit">Get in touch</button>
+                </li>
+              </ul>
+            </form>
+          </li>
+        </ul>
```
Comment (anagudelogu) [inline index.html]: - [x] Great job working on this section! 🚀 However, I think that using a `<ul>` tag here is not semantically correct as these elements don't represent a list. I suggest you remove the list and keep the section as your flex container for the paragraph and the form! 🤓 🔧

File: index.html [hunk 2]
```diff
@@ -351,6 +351,61 @@ <h4 class="featureText">Terminal</h4>
...
+                  <input type="email" name="email" placeholder="Email address" required>
+                </li>
+                <li class="formElement">
+                  <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+                </li>
+                <li id="submitForm">
+                  <button class="button" id="subBut" type="submit">Get in touch</button>
+                </li>
+              </ul>
+            </form>
```
Comment (anagudelogu) [inline index.html]: - [x] You did an excellent job using `Formspree` to collect data! 🥇 I would only suggest that you style a bit the `<textarea>` so it matches the Figma design:

|Template|Your design|
|---|---|
| image | Screenshot from 2023-01-17 16-34-05 |


As you can see, your `<textarea>` is half the height of the Figma design, please fix it as well as the color of the social media icons! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @evillaz, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by evillaz** [comment]
@anagudelogu Hello and thank you for reviewing. I have one question, is it ok if I use ul for the elements inside the form like this?
ul

File: index.html [hunk 3]
```diff
@@ -351,6 +351,55 @@ <h4 class="featureText">Terminal</h4>
...
+          <ul class="flex-container" id="formElements">
+            <li class="formElement">
+              <input class="fElement" type="text" name="name" placeholder="Full name" maxlength="30" required>
+            </li>
+            <li class="formElement">
+              <input type="email" name="email" placeholder="Email address" required>
+            </li>
+            <li class="formElement">
+              <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+            </li>
```
Comment (GedeonTS) [inline index.html]: - [x] As requested by the previous reviewer [here](https://github.com/evillaz/my-portfolio/pull/4#discussion_r1072856120), kindly remove the unordered list syntax to make your project semantically correct:

here is how the code might be :+1: 

```
<form>
<input/>
<input/>
<textarea ></textarea>
<button>Get in touch</button>
</form>


```

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @evillaz , 

Good job implementing some of the changes suggested by the previous reviewer!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- Good PR message  :+1: 
- Good UI design :+1: 
- No linter errors :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @evillaz  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Form is working as expected
✔️ Very descriptive PR heading and summary
✔️ All linters check are passing


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**